backend/vcdat/FlaskTests.py

```diff
@@ -52,6 +52,18 @@ def test_getTemplates(self):
         result = self.app.get('/getTemplates')
         self.assertNotEqual(result.data, "[]")
 
+    def test_browseFiles(self):
+        folder = "emptyFolder"
+        os.mkdir(folder)
+        result = self.app.get('/browseFiles', query_string='path=' + folder)
+        os.rmdir(folder)
+        self.assertNotEqual(result.data, "[]")
```

Comment:
what should be returned if a folder is empty? Can we test for the correct result as well? self.assertEqual(result.data, "????")